### ISSUE TITLE
Object is not same place in two hololens

### ISSUE THREAD

**Issue by DeenuMaurya** [KeanW/Dancing-Robot#1]
hello ,

         I  m share successfully in my two hololens but both hololens object is not see same place

**Comment by KeanW** [comment]
Yes, this is a known issue. Feel free to fix and submit a pull request, if you can find the solution (I stopped looking into it after a few weeks).
